### DISCUSSION TITLE
pyaedt_with_IDE.bat as part of release?

### DISCUSSION THREAD

**Discussion by hui-zhou-a** [ansys/pyaedt#1297] [Ideas]
Is it a good idea to make pyaedt_with_IDE.bat as part of release? In this case, customer can download it standalone instead of the whole package?

@maxcapodi78 @Samuelopez-ansys

**Comment by maxcapodi78** [comment]
@ring630  It is already part of the documentation. you can find it here
https://aedtdocs.pyansys.com/Resources/Installation.html

**Comment by Samuelopez-ansys** [comment]
Lock discussion. The bat was excluded from the release and the user can access from the documentation.
